bookrack-frontend/src/App.tsx

```diff
@@ -19,13 +20,27 @@ function App() {
      */
 
     const loadPastReads = (): Book[] => {
-        const savedPastReads = localStorage.getItem('pastReads');
-        return savedPastReads ? JSON.parse(savedPastReads) : [];
+        const savedPastReads = localStorage.getItem("pastReads")
+        return savedPastReads ? JSON.parse(savedPastReads) : []
     }
 
-
     const [pastReads, setPastReads] = useState<Book[]>(loadPastReads);
 
+    /**
+     * Loads the list of future reads from the local storage.
+     *
+     * @return {Book[]} - An array of the Book objects (parsed savedFutureReads) if it exists,
+     * otherwise it returns an empty array
```

Comment:
You might want to delete the comments.

Reply:
They are docstrings. I left them there because they are useful for understanding the code.